exporter/dynatraceexporter/internal/serialization/histogram.go

```diff
@@ -80,18 +85,26 @@ func estimateHistMinMax(dp pmetric.HistogramDataPoint) (float64, float64) {
 
 	var min, max float64
 
-	// Use lower bound for min unless it is the first bucket which has no lower bound, then use upper
-	if minIdx == 0 {
-		min = bounds[minIdx]
+	if dp.HasMin() {
+		min = dp.Min()
 	} else {
-		min = bounds[minIdx-1]
+		// Use lower bound for min unless it is the first bucket which has no lower bound, then use upper
```

Comment:
Shouldn't we consider the case when there's only one bucket (`(-Inf, Inf)`) and it doesn't contain any value? Like we do in the Java exporter? https://github.com/dynatrace-oss/opentelemetry-metric-java/blob/main/dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/Serializer.java#L278